app/controllers/bulk_actions/bulk_action_application_controller.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+module BulkActions
+  # Base controller for bulk actions.
+  class BulkActionApplicationController < ApplicationController
```

Comment:
Extract base controller for bulk actions. Probably more to come as additional bulk actions are implemented ...

Reply:
Nice.